SPECS-EXTENDED/libguestfs/libguestfs.spec

```diff
@@ -1230,6 +1181,17 @@ rm ocaml/html/.gitignore
 
 
 %changelog
+* Thu Jan 20 2022 Thomas Crain <thcrain@microsoft.com> - 1.44.0-4
```

Comment:
Did we publish 1.44.0-3? If not, I suggest merging both versions. It would look weird when we provide 1.44.0-2 and 1.44.0-4 skipping release `-3`.

Reply:
Sure, I'll merge these.